multimedia/VLC/Portfile

```diff
@@ -76,15 +96,15 @@ build               {}
 
 destroot {
     xinstall -m 755 -d ${destroot}${applications_dir}
-    system "ditto --rsrc ${workpath}/VLC-${version}/VLC.app ${destroot}${applications_dir}/VLC.app"
+    system "ditto --rsrc ${workpath}/VLC-${version}/VLC.app ${destroot}${applications_dir}/VLC3.app"
```

Comment:
I personally find it a bit strange to name the app VLC3.app.

Is there a case where someone would like to have VLC2 and VLC3 installed at the same time?

Reply:
> Is there a case where someone would like to have VLC2 and VLC3 installed at the same time?

IMO highly unlikely, but possibly for testing purposes (?).

Reply:
Couldn't testing be done by switching VLC version using `port deactivate`/`port activate`?

Reply:
It can, just conflict should be set in the portfile.